src/services/vuestic-ui/themes.ts

```diff
@@ -14,67 +14,21 @@ export const COLOR_THEMES = [
     colors: {
       primary: '#154ec1',
       secondary: '#767c88',
-      background: '#f6f7f6',
       success: '#3d9209',
       info: '#2c82e0',
       danger: '#e42222',
       warning: '#ffd43a',
-      white: '#ffffff',
-      dark: '#262824',
-      gray: '#767c88',
-    },
-    components: {
-      VaNavbar: {
-        color: 'white',
-        textColor: 'dark',
-      },
-      VaSidebar: {
-        color: 'white',
-        textColor: 'dark',
-      },
-      VaCard: {
-        color: 'white',
-      },
-      VaCardContent: {
-        textColor: 'dark',
-      },
-      VaCardTitle: {
-        textColor: 'dark',
-      },
     },
   },
   {
     name: THEME_NAMES.DARK,
     colors: {
       primary: '#6F92DA',
       secondary: '#A8AEBA',
-      background: '#2f302d',
       success: '#3d9209',
       info: '#2c82e0',
       danger: '#e42222',
       warning: '#ffd43a',
```

Comment:
We lost here dark background colors. So dark theme isn't really dark now. Consider adding here backgroundPrimary, backgroundSecondary etc. colors.